contrib/completion/bash/docker

```diff
@@ -4371,7 +4371,7 @@ _docker_node_ls() {
 
 	case "$prev" in
 		--filter|-f)
-			COMPREPLY=( $( compgen -W "id label membership name role" -S = -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "id label membership name node.label role" -S = -- "$cur" ) )
```

Comment:
We need to add a
```bash
     node.label)
        return
        ;;
```
branch in the preceding case statement in order to silence the following completions. Otherwise `docker node ls --filter node.label=-` would suggest `--filter  --format  --help    --quiet   -f        -q `.

Reply:
Thanks for review! ❤️ I updated the PR; made it `label|node.label)` (as I expect `label` currently had the same issue)

Reply:
Ah,  yes, you are right!